exdb/templates/exdb/search_results.html

```diff
@@ -0,0 +1,62 @@
+{% extends "exdb/base.html" %}
+{% load i18n %}
+{% load exdbtags %}
+{% load staticfiles %}
+{% block css %}
+    {{ block.super }}
+    <link rel="stylesheet" type="text/css" href="{% static 'exdb/css/search_results.css' %}" />
+{% endblock %}
+{% block javascript %}
+    {{ block.super }}
+    <script type="text/javascript" src="{% static 'exdb/js/search_results.js' %}"></script>
+{% endblock %}
+{% block content %}
+    {{ block.super }}
+
+    <h1>{% trans "Search Results" %}</h1>
+    <p><a href="{% url 'home' %}">{% trans "Home" %}</a></p>
+
+    {% if experiences %}
+        <table id="search-results" class="tablesorter">
+            <thead>
+                <tr>
+                    <th>{% trans "Experience Name" %}</th>
+                    <th>{% trans "Planners" %}</th>
+                    <th>{% trans "Status" %}</th>
+                    <th>{% trans "Building" %}</th>
+                    <th>{% trans "Keywords" %}</th>
+                    <th data-sorter="shortDate">{% trans "Start Time" %}</th>
+                    <th data-sorter="shortDate">{% trans "End Time" %}</th>
```

Comment:
We should have Type & Subtype as columns in this table. I'm pretty sure Travis will want to filter by them.
In order to save space, we can remove the End Time column since its usefulness here is limited.